Rocket.Chat.ShareExtension/State/SEState.swift

```diff
@@ -42,7 +42,7 @@ struct SEState {
         let displayedRooms: [Subscription]
         switch searchRooms {
         case .none:
-            displayedRooms = rooms
+            displayedRooms = rooms.filter { $0.open }
         case .searching(let search):
             let search = search.lowercased()
             displayedRooms = rooms.filter {
```

Comment:
What about the `case .started:`? Do we need to add it here too?

Reply:
Actually, yeah.